Increase wait time to restore flows after MW ends in test_085_end_running_mw_on_switch

Hi,

Eventually we have the following test failing:
```
300_____________ TestE2EMaintenance.test_085_end_running_mw_on_switch _____________
301self = <tests.test_e2e_15_maintenance.TestE2EMaintenance object at 0x7f6d143191c0>
302    def test_085_end_running_mw_on_switch(self):
303        self.restart_and_create_circuit()
304    
305        # Sets up the maintenance window information
306        mw_start_delay = 30
307        mw_duration = 60
308        start = datetime.now() + timedelta(seconds=mw_start_delay)
309        end = start + timedelta(seconds=mw_duration)
310    
311        # Sets up the maintenance window data
312        payload = {
313            "description": "mw for test 85",
314            "start": start.strftime(TIME_FMT),
315            "end": end.strftime(TIME_FMT),
316            "items": [
317                "00:00:00:00:00:00:00:02"
318            ]
319        }
320    
321        # Creates a new maintenance window
322        api_url = KYTOS_API + '/maintenance'
323        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
324        data = response.json()
325    
326        # Extracts the maintenance window id from the JSON structure
327        mw_id = data["mw_id"]
328    
329        # Gets the maintenance schema
330        api_url = KYTOS_API + '/maintenance/' + mw_id
331        response = requests.get(api_url)
332        assert response.status_code == 200, response.text
333        json_data = response.json()
334        assert json_data['id'] == mw_id
335    
336        # Waits for the MW to start
337        time.sleep(mw_start_delay + 5)
338    
339        # Verifies the flow behavior during the maintenance
340        s2 = self.net.net.get('s2')
341        flows_s2 = s2.dpctl('dump-flows')
342        assert 'dl_vlan=100' not in flows_s2
343        assert len(flows_s2.split('\r\n ')) == 1
344    
345        # Checks connectivity during maintenance
346        h11, h3 = self.net.net.get('h11', 'h3')
347        h11.cmd('ip link add link %s name vlan100 type vlan id 100' % (h11.intfNames()[0]))
348        h11.cmd('ip link set up vlan100')
349        h11.cmd('ip addr add 100.0.0.11/24 dev vlan100')
350        h3.cmd('ip link add link %s name vlan100 type vlan id 100' % (h3.intfNames()[0]))
351        h3.cmd('ip link set up vlan100')
352        h3.cmd('ip addr add 100.0.0.2/24 dev vlan100')
353        result = h11.cmd('ping -c1 100.0.0.2')
354        assert ', 0% packet loss,' in result
355    
356        # Ends the maintenance window information
357        api_url = KYTOS_API + '/maintenance/' + mw_id + '/end'
358        end_response = requests.patch(api_url)
359        assert end_response.status_code == 200
360    
361        # Waits just a couple of seconds to give kytos enough time to restore the flows
362        time.sleep(3)
363    
364        # Verifies the flow behavior and connectivity after ending the maintenance
365        flows_s2 = s2.dpctl('dump-flows')
366        assert len(flows_s2.split('\r\n ')) == 3
367        result = h11.cmd('ping -c1 100.0.0.2')
368>       assert ', 0% packet loss,' in result
369E       AssertionError: assert ', 0% packet loss,' in 'PING 100.0.0.2 (100.0.0.2) 56(84) bytes of data.\r\n\r\n--- 100.0.0.2 ping statistics ---\r\n1 packets transmitted, 0 received, 100% packet loss, time 0ms\r\n\r\n'
370tests/test_e2e_15_maintenance.py:886: AssertionError
371=============================== warnings summary ===============================
372
```

Looking for the logs of this execution, we can see the following messages:

```
Jul 18 06:51:15 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:15,928 (Thread-91) werkzeug:INFO _internal:113:  127.0.0.1 - - [18/Jul/2022 06:51:15] "#033[37mPATCH /api/kytos/maintenance/b759dcde05d242cf97a32d5df73c16f1/end HTTP/1.1#033[0m" 200 -
Jul 18 06:51:15 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:15,928 (MainThread) kytos.core.controller:DEBUG controller:526:  looking for listeners for kytos/maintenance.end_switch
Jul 18 06:51:15 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:15,929 (MainThread) kytos.core.controller:DEBUG controller:536:  Calling listeners for kytos/maintenance.end_switch
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,185 (Thread-96) kytos.napps.kytos/flow_manager:INFO main:536:  Send FlowMod from request dpid: 00:00:00:00:00:00:00:03, command: delete, force: True, flows_dict: {'flows': [{'cookie': 12289618010867041865, 'cookie_mask': 18446744073709551615}], 'force': True}
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,303 (thread_pool_sb_1) kytos.napps.kytos/flow_manager:INFO main:376:  Consistency check: alien flow on switch 00:00:00:00:00:00:00:03
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,330 (thread_pool_sb_1) kytos.napps.kytos/flow_manager:INFO main:381:  Flow forwarded to switch 00:00:00:00:00:00:00:03 to be deleted. Flow: {'flows': [{'switch': '00:00:00:00:00:00:00:03', 'table_id': 0, 'match': {'in_port': 1, 'dl_vlan': 100}, 'priority': 32768, 'idle_timeout': 0, 'hard_timeout': 0, 'cookie': 12289618010867041865, 'id': '59969a575f28f20572ed31732025f6c3', 'stats': {'byte_count': 520, 'duration_sec': 6, 'duration_nsec': 648000000, 'packet_count': 6}, 'cookie_mask': 0, 'instructions': [{'instruction_type': 'apply_actions', 'actions': [{'vlan_id': 100, 'action_type': 'set_vlan'}, {'action_type': 'push_vlan', 'tag_type': 's'}, {'vlan_id': 593, 'action_type': 'set_vlan'}, {'port': 3, 'action_type': 'output'}]}]}]}
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,337 (thread_pool_sb_1) kytos.napps.kytos/flow_manager:INFO main:376:  Consistency check: alien flow on switch 00:00:00:00:00:00:00:03
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,341 (Thread-97) kytos.napps.kytos/flow_manager:INFO main:536:  Send FlowMod from request dpid: 00:00:00:00:00:00:00:01, command: delete, force: True, flows_dict: {'flows': [{'cookie': 12289618010867041865, 'cookie_mask': 18446744073709551615}], 'force': True}
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,376 (thread_pool_sb_1) kytos.napps.kytos/flow_manager:INFO main:381:  Flow forwarded to switch 00:00:00:00:00:00:00:03 to be deleted. Flow: {'flows': [{'switch': '00:00:00:00:00:00:00:03', 'table_id': 0, 'match': {'in_port': 3, 'dl_vlan': 593}, 'priority': 32768, 'idle_timeout': 0, 'hard_timeout': 0, 'cookie': 12289618010867041865, 'id': '723c29ca053037e6fa3e704902035a41', 'stats': {'byte_count': 446, 'duration_sec': 6, 'duration_nsec': 632000000, 'packet_count': 5}, 'cookie_mask': 0, 'instructions': [{'instruction_type': 'apply_actions', 'actions': [{'action_type': 'pop_vlan'}, {'port': 1, 'action_type': 'output'}]}]}]}
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,482 (thread_pool_app_3) kytos.napps.kytos/mef_eline:INFO evc:141:  EVC 8d7e77baf90649 was synced to the storehouse.
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,490 (thread_pool_app_3) kytos.napps.kytos/mef_eline:DEBUG evc:490:  EVC(8d7e77baf90649, my evc1) will be deployed.
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,822 (Thread-98) kytos.napps.kytos/flow_manager:INFO main:536:  Send FlowMod from request dpid: 00:00:00:00:00:00:00:02, command: add, force: False, flows_dict: {'flows': [{'match': {'in_port': 2, 'dl_vlan': 2061}, 'cookie': 12289618010867041865, 'actions': [{'action_type': 'set_vlan', 'vlan_id': 4038}, {'action_type': 'output', 'port': 3}]}, {'match': {'in_port': 3, 'dl_vlan': 4038}, 'cookie': 12289618010867041865, 'actions': [{'action_type': 'set_vlan', 'vlan_id': 2061}, {'action_type': 'output', 'port': 2}]}], 'force': False}
Jul 18 06:51:17 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:17,971 (Thread-99) kytos.napps.kytos/flow_manager:INFO main:536:  Send FlowMod from request dpid: 00:00:00:00:00:00:00:01, command: add, force: False, flows_dict: {'flows': [{'match': {'in_port': 1, 'dl_vlan': 100}, 'cookie': 12289618010867041865, 'actions': [{'action_type': 'set_vlan', 'vlan_id': 100}, {'action_type': 'push_vlan', 'tag_type': 's'}, {'action_type': 'set_vlan', 'vlan_id': 2061}, {'action_type': 'output', 'port': 3}]}, {'match': {'in_port': 3, 'dl_vlan': 2061}, 'cookie': 12289618010867041865, 'actions': [{'action_type': 'pop_vlan'}, {'action_type': 'output', 'port': 1}]}], 'force': False}
Jul 18 06:51:18 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:18,171 (Thread-100) kytos.napps.kytos/flow_manager:INFO main:536:  Send FlowMod from request dpid: 00:00:00:00:00:00:00:03, command: add, force: False, flows_dict: {'flows': [{'match': {'in_port': 1, 'dl_vlan': 100}, 'cookie': 12289618010867041865, 'actions': [{'action_type': 'set_vlan', 'vlan_id': 100}, {'action_type': 'push_vlan', 'tag_type': 's'}, {'action_type': 'set_vlan', 'vlan_id': 4038}, {'action_type': 'output', 'port': 2}]}, {'match': {'in_port': 2, 'dl_vlan': 4038}, 'cookie': 12289618010867041865, 'actions': [{'action_type': 'pop_vlan'}, {'action_type': 'output', 'port': 1}]}], 'force': False}
Jul 18 06:51:18 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:18,215 (thread_pool_sb_3) kytos.napps.kytos/flow_manager:INFO main:376:  Consistency check: alien flow on switch 00:00:00:00:00:00:00:01
Jul 18 06:51:18 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:18,255 (thread_pool_sb_3) kytos.napps.kytos/flow_manager:INFO main:381:  Flow forwarded to switch 00:00:00:00:00:00:00:01 to be deleted. Flow: {'flows': [{'switch': '00:00:00:00:00:00:00:01', 'table_id': 0, 'match': {'in_port': 1, 'dl_vlan': 100}, 'priority': 32768, 'idle_timeout': 0, 'hard_timeout': 0, 'cookie': 12289618010867041865, 'id': 'c3c9fcdbca515680be7cb6ecc664901b', 'stats': {'byte_count': 426, 'duration_sec': 6, 'duration_nsec': 925000000, 'packet_count': 5}, 'cookie_mask': 0, 'instructions': [{'instruction_type': 'apply_actions', 'actions': [{'vlan_id': 100, 'action_type': 'set_vlan'}, {'action_type': 'push_vlan', 'tag_type': 's'}, {'vlan_id': 593, 'action_type': 'set_vlan'}, {'port': 4, 'action_type': 'output'}]}]}]}
Jul 18 06:51:18 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:18,259 (thread_pool_sb_3) kytos.napps.kytos/flow_manager:INFO main:376:  Consistency check: alien flow on switch 00:00:00:00:00:00:00:01
Jul 18 06:51:18 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:18,295 (thread_pool_app_3) kytos.napps.kytos/mef_eline:INFO evc:141:  EVC 8d7e77baf90649 was synced to the storehouse.
Jul 18 06:51:18 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:18,296 (thread_pool_app_3) kytos.napps.kytos/mef_eline:INFO evc:552:  EVC(8d7e77baf90649, my evc1) was deployed.
Jul 18 06:51:18 runner-rmtk68xb-project-107-concurrent-0 2022-07-18 06:51:18,296 (thread_pool_sb_3) kytos.napps.kytos/flow_manager:INFO main:381:  Flow forwarded to switch 00:00:00:00:00:00:00:01 to be deleted. Flow: {'flows': [{'switch': '00:00:00:00:00:00:00:01', 'table_id': 0, 'match': {'in_port': 4, 'dl_vlan': 593}, 'priority': 32768, 'idle_timeout': 0, 'hard_timeout': 0, 'cookie': 12289618010867041865, 'id': '45e376d19c24feddd9f1687adcf52074', 'stats': {'byte_count': 544, 'duration_sec': 6, 'duration_nsec': 906000000, 'packet_count': 6}, 'cookie_mask': 0, 'instructions': [{'instruction_type': 'apply_actions', 'actions': [{'action_type': 'pop_vlan'}, {'port': 1, 'action_type': 'output'}]}]}]}
```

Analyzing those messages, everything looks okay (despite the unnecessary consistency check removal, but that is okay since it is a delete_strict). The only thing that doesn't look good is the timing frame. The time taken by kytos since the `PATCH /maintenance/xxx/end` to the actual redeployment of th EVC was almost 3 seconds. And 3 seconds is the same time we wait on the test.

Thus I believe we should increase the wait time in the test to something like 5 seconds, to give Kytos more time to redeploy the EVC upon finishing the MW.

--- a/tests/test_e2e_15_maintenance.py
+++ b/tests/test_e2e_15_maintenance.py
@@ -877,7 +877,7 @@ class TestE2EMaintenance:
         assert end_response.status_code == 200
 
         # Waits just a couple of seconds to give kytos enough time to restore the flows
-        time.sleep(4)
+        time.sleep(5)
 
         # Verifies the flow behavior and connectivity after ending the maintenance
         flows_s2 = s2.dpctl('dump-flows')

--- a/tests/test_e2e_15_maintenance.py
+++ b/tests/test_e2e_15_maintenance.py
@@ -877,7 +877,7 @@ class TestE2EMaintenance:
         assert end_response.status_code == 200
 
         # Waits just a couple of seconds to give kytos enough time to restore the flows
-        time.sleep(3)
+        time.sleep(4)
 
         # Verifies the flow behavior and connectivity after ending the maintenance
         flows_s2 = s2.dpctl('dump-flows')

